persistent/Database/Persist/Sql/Migration.hs

```diff
@@ -294,13 +306,22 @@ newtype PersistUnsafeMigrationException
 instance Show PersistUnsafeMigrationException where
   show (PersistUnsafeMigrationException mig) =
     concat
-      [ "\n\nDatabase migration: manual intervention required.\n"
-      , "The unsafe actions are prefixed by '***' below:\n\n"
-      , unlines $ map displayMigration mig
+      [ "\n\nDatabase migration: manual intervention required.\n",
+        "The unsafe actions are prefixed by '***' below:\n\n",
+        unlines $ map displayMigration mig
       ]
     where
       displayMigration :: (Bool, Sql) -> String
-      displayMigration (True,  s) = "*** " ++ unpack s ++ ";"
+      displayMigration (True, s) = "*** " ++ unpack s ++ ";"
       displayMigration (False, s) = "    " ++ unpack s ++ ";"
 
 instance Exception PersistUnsafeMigrationException
+
+-- | Create an index for the given 'EntityField' and table.
+--
+-- @since 2.13.3.0
```

Comment:
```suggestion
-- | Create an index for the given 'EntityField' and table.
--
-- This function returns a 'Migration', and so can be sequenced with another migration in a `do` block.
--
-- @ 
-- migrations :: 'Migration'
-- migrations = do
--     migrateAll
--     createSearchIndex UserName
-- @
--
-- @since 2.13.3.0
```